src/main/java/com/atlassian/bitbucket/jenkins/internal/credentials/JenkinsToBitbucketCredentials.java

```diff
@@ -14,15 +15,25 @@
     /**
      * Converts the input credential id in Bitbucket Credentials.
      *
-     * @param credentialId, credentials id.
+     * @param credentialId the credentials id
      * @return Bitbucket credentials
      */
     BitbucketCredentials toBitbucketCredentials(@Nullable String credentialId);
 
+    /**
+     * Converts the input credential id for the given context into Bitbucket Credentials.
+     *
+     * @param credentialId the credentials id
+     * @param context      the {@link Item context} to retrieve the credentials from
+     * @return Bitbucket credentials
+     */
```

Comment:
`@Since` maybe? I know we are not super diligent and good about it, but I do prefer that we add them when we make changes